pkg/cmd/server/origin/openshift_apiserver.go

```diff
@@ -76,9 +77,10 @@ import (
 )
 
 type OpenshiftAPIExtraConfig struct {
-	KubeClientInternal    kclientsetinternal.Interface
-	KubeletClientConfig   *kubeletclient.KubeletClientConfig
-	KubeInternalInformers kinternalinformers.SharedInformerFactory
+	CoreAPIServerClientConfig *restclient.Config
```

Comment:
What's different to the GenericConfig loopback config? What is the "CoreAPIServer"? This needs a better name or at the verry least a godoc.

Reply:
> What's different to the GenericConfig loopback config? What is the "CoreAPIServer"? This needs a better name or at the verry least a godoc.

This is a connection back to the kube-apiserver, NOT back to the openshift-apiserver.  I'll try to think of a better name.

Reply:
KubeLoopbackClientConfig ?